solutions/standard/variables.tf

```diff
@@ -38,6 +38,12 @@ variable "plan" {
   default     = "platinum"
 }
 
+variable "existing_db_instance_crn" {
+  type        = string
+  default     = null
+  description = "The CRN of an existing Elasticsearch instance. If not specified, a new instance is created."
```

Comment:
Can we make this more verbose. I think the version is a good template.

The CRN of an existing Databases for Elasticsearch instance. If no value is specified, a new instance is created.